sdcm/sct_config.py

```diff
@@ -228,18 +224,18 @@ class SCTConfiguration(dict):
                   Time in minutes, Time of execution for stress commands from stress_cmd parameters
                   and is used in test duration calculation
              """),
-        dict(name="n_db_nodes", env="SCT_N_DB_NODES", type=int_or_list,
+        dict(name="n_db_nodes", env="SCT_N_DB_NODES", type=int_or_comma_separated_int,
```

Comment:
but we use format like: `n_db_nodes: '5 5'` - it's not int nor comma separated int... shouldn't be `int_or_space_separated_int`?

Reply:
yeah, it should be space separated

Reply:
changed the name (in code, docs, commit/PR)